src/rdf4cpp/rdf/Literal.cpp

```diff
@@ -664,7 +666,7 @@ Literal Literal::as_language_tag_eq(Literal const &other, Node::NodeStorage &nod
     }                                                  \
     RDF4CPP_DETAIL_TRY_WRITE_STR("\"");
 
-bool Literal::serialize(void *const buffer, writer::Cursor *cursor, writer::FlushFunc const flush) const noexcept {
+bool Literal::serialize(void *const buffer, writer::Cursor *cursor, writer::FlushFunc const flush, bool short_form) const noexcept {
```

Comment:
This should be a template `template<bool short_form>`

Reply:
If i make this a template, it needs to go into the header. With all the makro stuff going on, that might be a bad idea.

Reply:
Doesn't need to go in the header, all instantiations will be in the cpp file if you put the other serialize functions in the cpp file as well instead of defining them in the header